### ISSUE TITLE
pmore hang

### ISSUE THREAD

**Issue by kcwu** [ptt/pttbbs#1]
How to reproduce
  use this https://github.com/kcwu/pttbbs-fuzzy

```
cd mbbsd
make pmore
echo '0000000: 0c 23 00 2c 66 31 23 0a 30 ab ab ab ab ab ab     .#.,f1#.0......' | xxd -r > input
./pmore input
```

detected by afl-fuzz

**Comment by wens** [comment]
@kcwu @hungte Could you guys review this?

**Comment by holishing** [comment]
I wonder if this issue is actually fixed by @IepIweidieng in #77

**Comment by IepIweidieng** [comment]
The hanging caused by the specific case mentioned is fixed in #77.

There may still be unknown cases which lead to infinite loops like this.

However, I believe that it would be better to close this issue than to keep the issue open until such cases are actually found.

**Comment by wens** [comment]
Thanks.
